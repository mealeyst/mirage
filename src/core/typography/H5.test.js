import React from 'react'
import 'jest-styled-components'

import { H5 } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) H5', () => {
  const createH5 = (props) => {
    return shallowWithTheme(<H5 {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createH5())
    .toMatchSnapshot()
  })
})
