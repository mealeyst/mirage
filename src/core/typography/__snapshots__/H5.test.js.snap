// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) H5 matching the snapshot 1`] = `
.c0 {
  -webkit-letter-spacing: 0.5px;
  -moz-letter-spacing: 0.5px;
  -ms-letter-spacing: 0.5px;
  letter-spacing: 0.5px;
  color: #00003C;
  font-family: "ff-din-web",Helvetica,Arial,sans-serif;
  font-size: 1.6rem;
  font-weight: 500;
  font-style: italic;
  line-height: 1;
  -webkit-letter-spacing: .075rem;
  -moz-letter-spacing: .075rem;
  -ms-letter-spacing: .075rem;
  letter-spacing: .075rem;
}

<h5
  className="c0"
  fontSize="1.6rem"
/>
`;
